docs/source/gbd2019_models/risk_risk_correlation/child_growth_risks/index.rst

```diff
@@ -104,21 +104,106 @@ Wasting and Stunting
+
+Notably, in GBD, there are LBWSG risk exposures at birth and among the early and late neonatal age groups only. For the child growth failure risks (wasting, stunting, and underweight), there are risk exposures for all age groups under five years, although risk effects for these risks apply to the post-neonatal and 1-4 age groups only.
+
+Links to documentation for relevant risk exposure pages include:
+
+- :ref:`GBD 2019 Low birthweight short gestation risk exposure <2019_risk_exposure_lbwsg>`
+
+- :ref:`GBD 2020 Wasting risk exposure <2020_risk_exposure_wasting_state_exposure>`
+
+- :ref:`GBD 2020 Stunting risk exposure <2020_risk_exposure_child_stunting>`
+
+Vivarium Modeling Strategy
+----------------------------
+
+The strategy for modeling risk-risk correlations related to child anthropometry in this document was developed for the needs of the :ref:`acute malnutrition treatment and prevention simulation <2019_concept_model_vivarium_ciff_sam>`. Different strategies may be more appropriate for different project needs and should be reavaluated when necessary.
+
 Birthweight and Wasting
 ++++++++++++++++++++++++
 
+In GBD, the birthweight exposure distribution is tracked through the late neonatal age group (28 days after birth) and affects the risk of mortality during this period. For the :ref:`acute malnutrition treatment and prevention simulation <2019_concept_model_vivarium_ciff_sam>`, child wasting exposures are modeled as a dynamic transition model (:ref:`documented here <2020_risk_exposure_wasting_state_exposure>`) rather than a standard static propensity-based risk exposure model (:ref:`such as the stunting risk exposure model <2020_risk_exposure_child_stunting>`). Therefore, there will be two approaches to modeling the correlation and causal association between birthweight and wasting, including 1) the initialization of wasting risk exposure upon transition into the postneonatal age group based on a simulant's birthweight exposure value, and 2) the correlation between a simulant's propensity for wasting incidence over time ("x-factor") with their birth weight exposure.
+
+.. todo::
+
+   Link to x-factor documentation for wasting model when it is developed
+
 Correlation
 ^^^^^^^^^^^^
 
+Initialization
+~~~~~~~~~~~~~~~
+
+When simulants are initialized into the early or late neonatal age groups or born into the simulation, they will be assigned a birthweight exposure value. When they transition into the postneonatal age group, they should be initialized into a particular state in the wasting model (e.g. TMREL, mild wasting, moderate wasting, or severe wasting) *based on their birthweight percentile within their overall location- and sex-specific birthweight distribution among the late neonatal age group.*
+
+This should be done according to the methodology described in the :ref:`risk-risk correlation proposal page <2017_risk_models>`.
+
+.. todo::
+
+   Add more detail on the methodology here/as implemented in the BEP project
+
```

Comment:
This is something for us to think about more, perhaps for discussion and how to justify the birthweight and wasting causal relationship for BEP since the trials didnt show an effect on wasting/stunting. It possibly is a 'no evidence for does not mean evidence against'.  We might say something like although the BEP trials (with their limitations and weaknesses etc) did not show an effect of BEP on wasting/stunting at 6 months, we believe there is strong evidence that  birthweight has a causal effect on stunting/wasting. Hence in our model, we modelled this relationship.